src/humid.cc

```diff
@@ -97,7 +97,7 @@ tuple<size_t, size_t> readData(
  */
 size_t findHammingNeighbours(
     Trie<4, NLeaf> const& trie, size_t const distance, ofstream& log) {
-  time_t start {startMessage(log, "Calculating neighbours")};
+  size_t start = startMessage(log, "Calculating neighbours using Hamming distance");
```

Comment:
The return type of [`startMessage`](https://github.com/jfjlaros/HUMID/blob/8589285036651d83c06bc63e0d81b32787a5b898/src/log.h#L16) is `time_t`.

It is also good practice to [prefer the `{}`-initialiser syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax).